src/js/components/AppListComponent.jsx

```diff
@@ -433,16 +433,31 @@ var AppListComponent = React.createClass({
   },
 
   getInlineDialog: function (appNodes = []) {
-    var state = this.state;
+    var {props, state} = this;
```

Comment:
Mmm – we do that everywhere though. Let's discuss this "common practice" separately. Also, I've pushed some changes, please check again.